docs/admin/authorization.md

```diff
@@ -11,7 +11,7 @@ In Kubernetes, authorization happens as a separate step from authentication.
 See the [Accessing Control Overview](/docs/admin/accessing-the-api/) for an
 overview of how authentication and authorization are applied to requests.
 
-Authorization applies to all HTTP accesses on the main (secure) apiserver port.
+Authorization applies to all HTTP accesses on the main (secure) APIserver port.
```

Comment:
It's fine with "apiserver" in official document and please revert this kind of change.